eclipse-collections-code-generator/src/main/resources/impl/list/mutable/synchronizedPrimitiveList.stg

```diff
@@ -37,10 +37,14 @@ import org.eclipse.collections.impl.collection.mutable.primitive.AbstractSynchro
 import org.eclipse.collections.impl.factory.primitive.<name>Lists;
 import org.eclipse.collections.impl.lazy.primitive.Lazy<name>IterableAdapter;
 import org.eclipse.collections.impl.lazy.primitive.Reverse<name>Iterable;
+<if(primitive.specializedStream)>
+import java.util.Spliterator;<endif>
+
 
 /**
  * A synchronized view of a {@link Mutable<name>List}. It is imperative that the user manually synchronize on the collection when iterating over it using the
  * {@link <name>Iterator}, as per {@link Collections#synchronizedCollection(Collection)}.
+ * The same is true when using streams of the data by calling the {@link org.eclipse.collections.api.list.primitive.<name>List#primitveStream} and {@link org.eclipse.collections.api.list.primitive.<name>List#primitveParallelStream}
```

Comment:
The javadoc is also conditional based on whether a specializedStream exists. The natural way to say this would be in one sentence. "It is imperative that the user manually synchronize on the collection when iterating over it using an iterator or stream."

Reply:
Thanks. English is a mother tongue is a read advantage